src/Nacos.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Service = Ocelot.Values.Service;
+
+namespace Ocelot.Discovery.Nacos;
+
+public class Nacos : IServiceDiscoveryProvider
+{
+    private readonly INacosNamingService _client;
+    private readonly string _serviceName;
+    private readonly ILogger<Nacos> _logger;
```

Comment:
My concern is that we should adopt the standard Ocelot logger type used across all service discovery packages. Using an abstract logger may lead to potential support issues. Any logger can be wrapped by the Ocelot logger; therefore, we should switch to the native [Ocelot logger](https://github.com/search?q=repo%3AThreeMammals%2FOcelot%20%22private%20readonly%20IOcelotLogger%22&type=code) here.
```suggestion
    private readonly IOcelotLogger _logger;
```
Example in Consul:
https://github.com/ThreeMammals/Ocelot/blob/a8ba9848844b533181c68185098defc566a84728/src/Ocelot.Provider.Consul/Consul.cs#L12

Example in Kubernetes:
https://github.com/ThreeMammals/Ocelot/blob/a8ba9848844b533181c68185098defc566a84728/src/Ocelot.Provider.Kubernetes/Kube.cs#L21